### ISSUE TITLE
middlewares stripPrefixRegex is not

### ISSUE THREAD

**Issue by firstdorsal** [traefik/traefik#8801]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you do?

dynamic config file provider:
I tried to set this regex and did expect it to be parsed as one regex
```yaml
...
middlewares:
    pektin-proxy-strip-proxy:
        stripPrefixRegex:
            regex:
                - ^/proxy-[^/]{1,}
...              
```             

### What did you see instead?

Insted of parsing it as one regex it got parsed as two regexes because of the comma. I would expect this to happen if it was set as a string but not as an array. I also tried escaping it but with no success. The documentation for the stripPrefixRegex middleware does not cover this.

```json
{
    "stripPrefixRegex": {
        "regex": [
            "^/proxy-[^/]{1",
            "}"
        ]
    },
    "status": "enabled",
    "usedBy": [
        "pektin-proxy-crt@file",
        "pektin-proxy-gandi@file"
    ],
    "name": "pektin-proxy-strip-proxy@file",
    "provider": "file",
    "type": "stripprefixregex"
}
```

### What version of Traefik are you using?

traefik 2.6

### What is your environment & configuration?

Much config not regarding the problem

**Comment by ldez** [comment]
Hello,

I handle the problem inside our configuration parser, a PR will come soon.

Note: `[^/]{1,}` can be replaced by `[^/]+`

**Comment by traefiker** [comment]
Closed by #8850.
